Runtime/Properties/PropertyReflectionHelper.cs

```diff
@@ -39,7 +39,7 @@ public static List<LockablePropertyData> ExtractLockablePropertiesFromStep(IStep
             return result;
         }
 
-        public static List<LockablePropertyData> ExtractLockablePropertiesFromConditions(IConditionData data)
+        public static List<LockablePropertyData> ExtractLockablePropertiesFromConditions(IConditionData data, bool alsoRequired = true)
```

Comment:
Everything seems fine but the parameter `alsoRequired` seems a bit misleading to me, I don't understand what exactly is also required 🤔